### PR TITLE
Update install scripts to typescript

### PR DESCRIPTION
**Pull request by enzomonjardin**
We're updating the way we run our install scripts, requiring them to be re-written in typescript.

### CODE REVIEW

**Comment by enzomonjardin** [comment]
[Update Adalo components to be able to run in sandbox (and tweaks to packager as needed)](https://trello.com/c/XerqUPO2/1454-update-adalo-components-to-be-able-to-run-in-sandbox-and-tweaks-to-packager-as-needed)

**Review by adebroux — APPROVED** [review]
LGTM!

File: scripts/commons.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  }
+
+  return lines.join('\n')
+}
+
+export const getMapsApiKey = async (projectPath: string) => {
+  try {
+    const protonBundlePath = join(projectPath, 'proton-bundle.json')
+    const protonBundleText = await Deno.readTextFile(protonBundlePath)
+    const protonBundle = JSON.parse(protonBundleText)
```
Comment (guislain-adalo) [inline scripts/commons.ts:35]: Can this be done in one go?
`const module = await import(protonBundlePath, {
  with: { type: "json" },
});`

**Review by guislain-adalo — APPROVED** [review]

File: scripts/commons.ts (hunk 1 above)
Comment (enzomonjardin) [reply scripts/commons.ts:35]: Yuuuup

**Pull request merged**